.circleci/config.yml

```diff
@@ -38,67 +51,127 @@ commands:
+                  sudo installer -pkg AWSCLIV2.pkg -target /
+                name: Install .NET Core, AWS
+      - when:
+          condition:
+            equal: [ *linux, << parameters.os >>]
+          steps:
+            - run: 
+                command: |
+                  wget https://packages.microsoft.com/config/ubuntu/22.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
+                  sudo dpkg -i packages-microsoft-prod.deb
+                  rm packages-microsoft-prod.deb
+                  sudo apt update
+                  sudo apt install -y dotnet-sdk-6.0
+                name: Install .NET Core
       - run:
           command: |
             dotnet clean -c Release
             dotnet nuget locals all --clear
           working_directory: NineChronicles.Headless
           name: Clean Headless
       - run:
-          command: yarn concurrently npm:build-headless npm:bundle-player
-          shell: bash
+          command: yarn concurrently yarn:build-headless yarn:bundle-player
           name: Setup Software Components
-      - run:
+      - run: 
+          command: 'echo "::set-output name=apv::$(scripts/make-apv.sh)"'
+      - run: 
           command: |
-            if ($env:CIRCLE_BRANCH -eq "main") {
+            mkdir pack-dist
+            set -ex
+            mkdir -p dist
+            curl -o "dist/config.json" 'https://download.nine-chronicles.com/9c-launcher-config.json'
+          name: Copy config.json from the last release
+      - run: yarn sign-apv
+      - run: 
+          command: |
+            if [[ "$CIRCLE_BRANCH" == "main" ]]
+            then
               yarn release
-            } else {
+            else
               yarn run pack
-            }
-      - run: 
-          command: 7z a -r ../../pack-dist/Windows.zip *
-          working_directory: pack/Nine Chronicles-win32-x64/
+            fi
+      - when:
+          condition: 
+            equal: [ *windows, << parameters.os >>]
+          steps:
+          - run: 
+              command: 7z a -r ../../pack-dist/Windows.zip *
+              working_directory: pack/Nine Chronicles-win32-x64/
+      - when:
+          condition:
+            equal: [ *macos, << parameters.os >>]
+          steps:
+            - run:
+                command: |
+                  chmod +x "Nine Chronicles.app/Contents/Resources/app/9c.app/Contents/MacOS/9c"
+                  tar cvfz ../../pack-dist/macOS.tar.gz *
+                working_directory: pack/Nine Chronicles-darwin-x64/
+      - when:
+          condition:
+            equal: [ *linux, << parameters.os >>]
+          steps:
+            - run: 
+                command: |
+                  chmod +x resources/app/9c
+                  tar cvfz ../../pack-dist/Linux.tar.gz *
+                working_directory: pack/Nine Chronicles-linux-x64/
       - store_artifacts:
           path: pack-dist
           destination: dist-artifacts
-      - run:
+      - run: 
```

Comment:
Extra space. Probably running Prettier once would solve most of the styling issues.